demos/common/cpp/utils/include/utils/common.hpp

```diff
@@ -249,3 +250,69 @@ inline void logExecNetworkInfo(const InferenceEngine::ExecutableNetwork& execNet
         }
     }
 }
+
+inline
+void log_compiled_model_info(
```

Comment:
I think my way is better :)
https://github.com/openvinotoolkit/open_model_zoo/pull/2893/files#diff-746af3a0af0e254bedb0fa42f1604fc533a856185093fb9db870a2cf951e395bR236

Reply:
well, reviewed your implementation. I do not think we need involve templates to keep old func name `logExecNetworkInfo` with new API. Better to make clear difference from old API.

Reply:
I suggest to adhere camel case naming for functions, as we always do -` logCompiledModelInfo`.

Reply:
ok, renamed functions to logBasicModelInfo and logCompiledModelInfo, as change in style would be to massive for OMZ and not well justified.